FantasyUniverse/Customization/ViewModifiers.swift

```diff
@@ -0,0 +1,243 @@
+//
+//  ViewModifiers.swift
+//  FantasyUniverse
+//
+//  Created by TarasPeregrinus on 29.07.2022.
+//
+
+import SwiftUI
+import Combine
+import UIKit
+
+struct DefaultLabelStyle: ViewModifier {
+    
+    let width: CGFloat
+    let height: CGFloat
+    
+    func body(content: Content) -> some View {
+        content
+            .font(.appLargeFont)
+            .minimumScaleFactor(0.1)
+            .padding(.horizontal, 2)
+            .frame(width: width,
+                   height: height,
+                   alignment: .center)
+            .background(Color(Asset.Colors.aquamarine.color))
```

Comment:
It would be nice, although i have not found a way to do it. Asset.Colors is an enum that contains static lets, not cases, so You can not add Asset.Colors as a type for parameter in custom init. You will not be able to use dot notation to call .colorOne from inside a custom a background method, because colors are lets, not cases. We have faced the same issue on our previous project, while implementing Tuist (because it uses the same mechanism and almost the same generated code as SwiftGen does). If You know how to handle that - please let me know.